stack/keycloak/migration/02_add-client-pp.yml

```diff
@@ -7,8 +7,7 @@ changes:
       secret: ${SSO_CLIENT_SECRET}
       publicClient: false
       redirectUris:
-        - http://localhost:${DOCKER_PP_FRONTEND_PORT}/*
-        - http://localhost:${LOCAL_STANDALONE_FRONTEND_PORT}/*
+        - http://*
```

Comment:
```suggestion
        - *
```

Reply:
this was tried before, but didn't seem to work when looking into the keycloak console

Reply:
![grafik](https://github.com/it-at-m/Praktikumsplaner/assets/69314043/5472cb48-81d1-4487-93b0-a5d69cfe08df)
when trying the suggested solution, init-keycloak throws this error